### ISSUE TITLE
new forum post menu - not existing pref

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4909]
## Bug Description
This is shortcode for content of 
```
	function sc_post_content($parm = null)
	{

		$tp = e107::getParser();
		$pref = e107::getPref();
		$post = strip_tags($tp->toHTML($this->var['post_entry'], true, 'emotes_off, no_make_clickable', '', $pref['menu_wordwrap']));
		$post = $tp->text_truncate($post, varset($this->param['nfp_characters'], 120), varset($this->param['nfp_postfix'], '...'));

		return $post;
	}
```

`$pref['menu_wordwrap'] is NULL`

It is probably outdated code

**Comment by tgtje** [comment]
@Jimmi08 makes this sence ? 

define("PRFLAN_110", "words longer than the length entered will be wrapped onto a new line");
define("PRFLAN_111", "Wrap long words in menu text");`

**Comment by Jimmi08** [comment]
@tgtje yes, but is this still needed in responsive design? It should be wrapped to new line by css regardless this pref.  

**Or maybe I don't understand the purpose of this pref.**

My worst problem was - if you have post with link inserted, html tags are stripped, then full url left there...  and it was going out of place...  Max number of characters will help if you split this link... 

Will this pref help to this? If yes,  wrapped is not good word, maybe split into a new line? 

![image](https://user-images.githubusercontent.com/5429548/202032882-41fbb881-8dc7-409f-b451-a8ff00c1a288.png)

**Comment by tgtje** [comment]
Wording ??  overflow is general the same. Issue is : when url and design : nice if it follws the design, (breaks/overflows).
But it would be different if long lines/url's are in a printable form...  
Hard stuff : interesting link :[ this one](https://css-tricks.com/where-lines-break-is-complicated-heres-all-the-related-css-and-html/)
